compositor_render/src/transformations/web_renderer/chromium_sender_thread.rs

```diff
@@ -64,10 +61,23 @@ impl ChromiumSenderThread {
             return;
         };
 
+        let mut state = ThreadState {
+            browser,
+            shared_memories: HashMap::new(),
```

Comment:
```suggestion
            shared_memory: Vec::new(),
```

Reply:
I think it should be `HashMap<NodeId, Vec<..>`. Web renderer instance can be used by multiple nodes